Mathlib/Data/List/Defs.lean

```diff
@@ -410,6 +410,11 @@ def replaceIf : List α → List Bool → List α → List α
   | l, [], _ => l
   | n :: ns, tf :: bs, e@(c :: cs) => if tf then c :: ns.replaceIf bs cs else n :: ns.replaceIf bs e
 
+/-- `repeatList n l` is a list consisting of l repeated n times. -/
+
+@[simp]
+def repeatList (n: ℕ) (l: List α) : List α := (List.replicate n l).flatten
```

Comment:
Please put spaces around `:`, like this. (The same applies to a few other places below.)
```suggestion
def repeatList (n : ℕ) (l : List α) : List α := (List.replicate n l).flatten